### ISSUE TITLE
1.0.2 breaks encoding/decoding

### ISSUE THREAD

**Issue by fred-o** [havard/node-openid#161]
Release 1.0.2 breaks encoding/decoding on node.js versions older than 5.10. This is the stack trace we get:

```
> TypeError: binary is not a function
>   at Function.from (native)
>   at from (native)
>   at _base64encode (/usr/local/leget/.tmp/nar-leget-v4.5.1-2-gdedf650-bin-1474010725261/node_modules/openid/openid.js:97:10)
>   at _bigIntToBase64 (/usr/local/leget/.tmp/nar-leget-v4.5.1-2-gdedf650-bin-1474010725261/node_modules/openid/openid.js:106:10)
>   at Object.openid.associate (/usr/local/leget/.tmp/nar-leget-v4.5.1-2-gdedf650-bin-1474010725261/node_modules/openid/openid.js:608:35)
>   at chooseProvider (/usr/local/leget/.tmp/nar-leget-v4.5.1-2-gdedf650-bin-1474010725261/node_modules/openid/openid.js:816:16)
>   at /usr/local/leget/.tmp/nar-leget-v4.5.1-2-gdedf650-bin-1474010725261/node_modules/openid/openid.js:830:7
>   at /usr/local/leget/.tmp/nar-leget-v4.5.1-2-gdedf650-bin-1474010725261/node_modules/openid/openid.js:580:7
>   at /usr/local/leget/.tmp/nar-leget-v4.5.1-2-gdedf650-bin-1474010725261/node_modules/openid/openid.js:446:16
>   at Request._callback (/usr/local/leget/.tmp/nar-leget-v4.5.1-2-gdedf650-bin-1474010725261/node_modules/openid/openid.js:209:7)
>   at Request.self.callback (/usr/local/leget/.tmp/nar-leget-v4.5.1-2-gdedf650-bin-1474010725261/node_modules/request/request.js:187:22)
>   at emitTwo (events.js:100:13)
```

As far as we can tell, there is already an (undocumented) implementation of `Buffer.from`, which makes the polyfill not work as intended.

**Comment by havard** [comment]
1.0.1 had this problem. 1.0.2 was released to fix it and works fine for me on node 4.3.4 and 5.9.0. Are you sure you're using openid@1.0.2?

**Comment by fred-o** [comment]
My bad, it turns we had 1.0.1 after all. Sorry for the hubbub.

Cheers,
-- Fredrik
